### DISCUSSION TITLE
how does a self-hosted PDS server intergrate into the rest of bluesky?

### DISCUSSION THREAD

**Discussion by Gnomesenpai** [bluesky-social/atproto#2915] [General Discussion]
So as the title says, how does it work? I've been thinking about running my own instance as i like to self host, but not wrapped my head around how my little PDS servers would know and speak to the rest of the Bluesky community, I understand the ATProtocol is there but as a newbie I dont fully understand exactly how it works on the back end. 

Tl;Dr I want to host my own instance but still be able to see the wider bluesky app and my followers etc and for them to see my content still.

**Comment by bnewbold** [comment]
The short answer is that things will mostly "just work" from a self-hosted PDS. "Which PDS you are hosted on" is mostly not very visible or noticeable.

This paper (pre-print) gives an overview of the protocol: https://arxiv.org/abs/2402.03239
